TYPO3.Media/Classes/TYPO3/Media/Domain/Model/Thumbnail.php

```diff
@@ -24,9 +24,12 @@
  *
  * @Flow\Entity
  * @ORM\Table(
- *    indexes={
- * 		@ORM\Index(name="originalasset_configurationhash",columns={"originalasset", "configurationhash"})
- *    }
+ *  uniqueConstraints={
+ *      @ORM\UniqueConstraint(name="originalasset_configurationhash",columns={"originalasset", "configurationhash"})
```

Comment:
The name for this index (and the next one a few lines down) are not reflected in the DB migrations.